src/request_collector/server.py

```diff
@@ -24,26 +43,54 @@ def error_handler(context, exc_info):
     sys.exit()
 
 
+def setup_matrix(private_key: str) -> Tuple[GMatrixClient, Room]:
+    available_servers_url = DEFAULT_MATRIX_KNOWN_SERVERS[Environment.DEVELOPMENT]
+    available_servers = get_matrix_servers(available_servers_url)
+
+    def _http_retry_delay() -> Iterable[float]:
+        # below constants are defined in raiden.app.App.DEFAULT_CONFIG
+        return udp_utils.timeout_exponential_backoff(
+            DEFAULT_TRANSPORT_RETRIES_BEFORE_BACKOFF,
+            DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL / 5,
+            DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL,
+        )
+
+    client = make_client(
+        servers=available_servers,
+        http_pool_maxsize=4,
+        http_retry_timeout=40,
+        http_retry_delay=_http_retry_delay,
+    )
+    login_or_register(client, signer=LocalSigner(private_key=decode_hex(private_key)))
+
+    monitoring_room = join_global_room(
+        client=client,
+        name=MONITORING_BROADCASTING_ROOM,
```

Comment:
Notice this constant is just the suffix of the room name (the one that, on raiden, goes on `global_rooms` config and is accepted by `send_global`), e.g. `monitoring`, while the name accepted by `join_global_room` is the actual, qualified name, e.g. `raiden_ropsten_monitoring`. It's just a `'_'.join(['raiden', chain_name or chain_id, MONITORING_BROADCASTING_ROOM])`, but as the utils is more generic, it expects you to do this transform

Reply:
I'll split that into an utils as well, just to keep that synced